.gitlab-ci.yml

```diff
@@ -749,6 +749,7 @@ Installer:
           - rhos-01/rhel-8.7-nightly-x86_64
           - rhos-01/rhel-9.1-nightly-x86_64
           - rhos-01/centos-stream-9-x86_64
+          - rhos-01/fedora-37-x86_64
```

Comment:
Not supported on Fedora, see https://github.com/osbuild/osbuild-composer/pull/2878